Tests/CoreTests/ISO8601Tests.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+import XCTest
+@testable import Core
+
+class ISO8601Tests: XCTestCase {
+
+    static var allTests = [
+        ("testBasic", testBasic)
+    ]
```

Comment:
Listing the tests is necessary running them on Linux, Xcode can fortunately figure it out by itself. You're right, the other test file should declare its test too.